server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -0,0 +1,754 @@
+
+
+# LAYOUT.
+layout = html.Div([
+    html.Div(children=[
+        # Neighborhood Council Dashboard.
+        generate_summary_header(),
+        # Summary Dropdown.
+        generate_summary_dropdowns(),
+        html.Div(html.Br(), style={"height": "0.5vh"}),
+        # Line Chart for Number of Request throughout the day.
+        generate_summary_line_chart(),
+        html.Div(html.Br(), style=DIVIDER_STYLE),
+        html.Div(children=[
+            generate_summary_pie_chart(),
+            generate_summary_histogram()
+        ], style=merge_dict(EQUAL_SPACE_STYLE, {"width": "97.5vw"}))
+    ]),
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+    # Neighborhood Council Summarization Dashboard.
+    html.Div(children=[html.H2(COMPARISON_DASHBOARD_TITLE)],
+             style=merge_dict(CENTER_ALIGN_STYLE, {"height": "5vh"})),
+    # Comparison Dropdowns.
+    html.Div(children=[
+        generate_council_name_dropdown('nc_comp_dropdown'),
+        generate_council_name_dropdown('nc_comp_dropdown2')
+    ], style=merge_dict(EQUAL_SPACE_STYLE, {"width": "97.5vw", "height": "12vh"})),
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+    # NC Comparison - Indicator Visuals.
+    html.Div(children=[
+        html.Div(children=[
+            # Indicator Visuals for Total number of requests and the number of
+            # days the data spans across.
+            generate_comparison_total_req("total_req_card"),
+            generate_comparison_num_days("num_days_card")
+        ], style=merge_dict(EQUAL_SPACE_STYLE, {"width": "48.5vw"})),
+        # Indicator Visuals for Total number of requests and the number of days
+        # the data spans across.
+        html.Div(children=[
+            generate_comparison_total_req("total_req_card2"),
+            generate_comparison_num_days("num_days_card2")
+        ], style=merge_dict(EQUAL_SPACE_STYLE, {"width": "48.5vw"}))
+    ], style=merge_dict(EQUAL_SPACE_STYLE, {"width": "97.5vw"})),
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+    # NC Comparison -  Request Source Bar Charts.
+    html.Div(children=[
+        generate_comparison_req_source_bar("req_source_bar_chart"),
+        generate_comparison_req_source_bar("req_source_bar_chart2")
+    ], style=merge_dict(EQUAL_SPACE_STYLE, {"width": "97.5vw"})),
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+    # NC Comparison - Number of Requests per day Overlapping line chart.
+    generate_comparison_line_chart()
+])
+
+# CALLBACK FUNCTIONS.
+
+
+@callback(
+    [Output("nc_dropdown_filter", "options")],
+    Input("nc_dropdown", "value")
+)
+def generate_dynamic_filter(nc_dropdown):
+    """Enables the dashboard to show dynamic filters.
+
+    This function takes the selected neighborhood council (nc) value from the 
+    "nc_dropdown" dropdown and output a a list of available request types
+    from that neigbhorhood council.
+
+    Args:
+        nc_dropdown: A string argument automatically detected by Dash callback 
```

Comment:
I think something like `selected_nc` would be more accurate.

Reply:
Thanks for the suggestion - resolved